.github/workflows/wheel-builder.yml

```diff
@@ -123,6 +124,14 @@ jobs:
           - PYTHON: { VERSION: "pp311-pypy311_pp73" }
             MANYLINUX: { NAME: "manylinux_2_28_ppc64le", CONTAINER: "cryptography-manylinux_2_28:ppc64le", RUNNER: "ubuntu-24.04-ppc64le" }
 
+          # CFFI 2.0.0b is missing manylinux2014 wheels
```

Comment:
is this resolved in the final?

Reply:
Yup, I should be able to delete this, thanks for pointing this out.